BaseTools/Plugin/OverrideValidation/OverrideValidation.py

```diff
@@ -628,20 +657,61 @@ def path_parse():
     dummy_list = []
     pathtool = Edk2Path(Paths.WorkSpace, dummy_list)
 
-    # Use absolute module path to find package path
-    pkg_path = pathtool.GetContainingPackage(Paths.TargetPath)
-    rel_path = Paths.TargetPath[Paths.TargetPath.find(pkg_path):]
+    # check if we are asked to update an .inf file "in-place"
+    if (Paths.RegenPath is not None):
+        if (Paths.RegenPrefix is not None):
+            path_prefixes = Paths.RegenPrefix
+        else:
+            path_prefixes = []
+        path_prefixes.append (Paths.WorkSpace)
+
+        v1_regex = re.compile(r"#(Override|Track) : (.*?) \| (.*?) \| (.*?) \| (.*?)")
+        v2_regex = re.compile(r"#(Override|Track) : (.*?) \| (.*?) \| (.*?) \| (.*?) \| (.*?)")
+        with open (Paths.RegenPath) as fd:
+            RegenInfData = fd.read()
+
+        RegenInfOutData = ""
+        for line in RegenInfData.splitlines (True):
+            match = v1_regex.match(line)
+            if match is None:
+                match = v2_regex.match(line)
+
+            if match is not None:
+                rel_path = match.group(3)
+                abs_path = resolvePath(path_prefixes, rel_path)
```

Comment:
This function seems to be the same as GetAbsolutePathOnThisSystemFromEdk2RelativePath from `edk2toollib.uefi.edk2.path_utilities import Edk2Path`, might want to use the library function instead of a local implementation.

Reply:
Done.